mailpoet/assets/css/src/components-plugin/_forms.scss

```diff
@@ -218,3 +218,7 @@ progress::-moz-progress-bar {
     font-style: italic;
   }
 }
+
+.mailpoet-form-field-tags label.components-form-token-field__label {
```

Comment:
I haven't found how to disable the label in the Gutenberg component, but we need to hide it due to the different form design. The label doesn't fit our form, and we would have duplicity here.

Reply:
Aha. Thanks! That makes sense. Could you please add a similar comment also to the code? Maybe the component will change and then we will know why this was done and it will be safe to remove it.